### ISSUE TITLE
Basic Frog Navigation

### ISSUE THREAD

**Issue by MayorMonty** [MayorMonty/cpsc-4160#8]
User should be able to move frog around via keyboard navigation, including left, right, and jump.

- Create `frog` entity to represent current frog position and orientation
- Create `frog_control` and `frog_render` systems in `systems/`

**Comment by MayorMonty** [comment]
Progress Update: Basic motion works, now need to play with collisions with tetrominos. I have a basic implementation, but right now it's pretty janky.

Reading up online, separating out horizontal and vertical motion and collisions seems like it really helps
